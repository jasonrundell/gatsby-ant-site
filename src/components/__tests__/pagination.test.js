import React from 'react'
import renderer from 'react-test-renderer'

import Pagination from '../pagination'

describe('Pagination', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Pagination />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
