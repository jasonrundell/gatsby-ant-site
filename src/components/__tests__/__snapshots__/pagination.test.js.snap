// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination renders correctly 1`] = `
<nav
  aria-label="Pagination Navigation"
  role="navigation"
>
  <ul
    className=""
  >
    <li
      className=""
    >
      <a
        href="/"
      >
        Previous
      </a>
    </li>
    <li
      className=""
    >
      <a
        href="/"
      >
        Next
      </a>
    </li>
  </ul>
</nav>
`;
